ad7768-1fmcz/ad77681.c

```diff
@@ -367,6 +632,297 @@ int32_t ad77681_set_mclk_div(struct ad77681_dev *dev,
 	return ret;
 }
 
+/**
+ * Set the VCM output.
+ * @param dev - The device structure.
+ * @param VCM_out - The VCM output voltage.
+ * 					Accepted values: AD77681_VCM_HALF_VCC
+ *									 AD77681_VCM_2_5V
+	  								 AD77681_VCM_2_05V
```

Comment:
nitpick: This header comment looks messy.

Reply:
Done.